### ISSUE TITLE
Does not work on NodeJS v12

### ISSUE THREAD

**Issue by luxalpa** [alexgorbatchev/eslint-import-resolver-typescript#93]
Minimal Repro: https://github.com/luxalpa/repro-eslint-resolver-bug/tree/main

Using NodeJS version 12.22.10 or 13.14.0 it throws an error:

```
repro-eslint-resolver-bug/src/main.ts
  1:26  error  Unable to resolve path to module 'fs/promises'  import/no-unresolved
```

on 14.2 and 16.14.0 it does not report this error.

I minimalized the reproduction to show this specific issue - it's possible that in doing so I might have removed other bits that are important to work correctly. I however tested 100 or so possible configurations without finding a single one that does not throw an error on v12 (while at the same time still being enabled at all).

Not sure if I'm doing something wrong or this is a bug. Not sure how to workaround this issue either :/

I reproduced this on Elementary OS and inside dockers `node:12-alpine` image.

**Comment by JounQin** [comment]
`is-core-module` is used by `resolve` and then used in this project.

See https://github.com/inspect-js/is-core-module/blob/main/core.json#L38

It says the range is `[">= 10 && < 10.1", ">= 14"]`, if you think it's wrong, PR there instead, thanks for reporting.

**Comment by luxalpa** [comment]
thanks a lot for that link. I wish this was mentioned on the Readme, as I was just assuming that this tool would simply follow the typings to `@types` instead of doing this kind of thing. At the very least I feel it could give a better error message (if that is possible). :sweat_smile:

**Comment by JounQin** [comment]
We ignore core module as intended, PR welcome for notice.
